.github/workflows/ios-build.yml

```diff
@@ -29,6 +29,10 @@ jobs:
         with:
           node-version: 18
           cache: 'yarn'
+      - name: setup-cocoapods
+        uses: maxim-lobanov/setup-cocoapods@v1
+        with:
+          version: 1.15.2
```

Comment:
This looks like something that should be in a separate PR.

Reply:
Sure, [here it is](https://github.com/software-mansion/react-native-gesture-handler/pull/2765). I will remove these lines after we merge it.